src/bosh-director/lib/bosh/director/api/controllers/tasks_configs_controller.rb

```diff
@@ -0,0 +1,28 @@
+require 'bosh/director/api/controllers/base_controller'
+
+module Bosh::Director
+  module Api::Controllers
+    class TasksConfigsController < BaseController
+      put '/', consumes: :json do
```

Comment:
let's also add 'get /task_configs' and make it run similarly to runtime_configs.